lcls_tools/superconducting/scLinac.py

```diff
@@ -260,6 +287,36 @@ def __init__(self, cavityNum, rackObject, ssaClass=SSA,
         
         self.stepper_temp_PV: PV = PV(self.pvPrefix + "STEPTEMP")
         self.detune_best_PV: PV = PV(self.pvPrefix + "DFBEST")
+        self.detune_rfs_PV: PV = PV(self.pvPrefix + "DF")
+        
+        self.ades_max_PV: PV = PV(self.pvPrefix + "ADES_MAX")
+        self.rf_permit_pv: str = self.pvPrefix + "RFPERMIT"
+        self.quench_latch_pv: str = self.pvPrefix + "QUENCH_LTCH"
+    
+    def auto_tune(self, des_detune=0, limit=10000):
```

Comment:
@s-aderhold can you help me sanity check this function? I want it to work for the parking procedure too (intentionally detuning to 10k)